udata_gouvfr/theme/templates/home.html

```diff
@@ -198,6 +198,59 @@ <h3 class="title ellipsis-dot">
       </div>
 </section>
 {% endif %}
+{% if forum_topics %}
+<section class="default forum-topics">
+      <div class="container forum-topics-container">
+            <div class="row">
+                <div class="col-xs-12">
+                    <h3><a href="{{ config.DISCOURSE_URL }}">{{ _('Latest forum discussions') }}</a></h3>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-xs-12">
+                    <table class="table table-hover">
+                        <thead>
+                            <tr>
+                                <th>{{ _('Title') }}</th>
+                                <th>{{ _('Users') }}</th>
+                                <th>{{ _('Created at') }}</th>
+                                <th class="text-center">{{ _('Replies') }}</th>
+                                <th class="text-center">{{ _('Views') }}</th>
+                                <th class="text-center">{{ _('Likes') }}</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for topic in forum_topics %}
+                            <tr>
+                                <td>
+                                    <a href="{{topic.url}}">
+                                        <span class="topic-title">{{topic.title}}</span>
+                                        <span class="label label-primary">{{topic.category}}</span>
+                                    </a>
+                                </td>
+                                <td>
+                                    <a href="{{topic.url}}">
+                                        {% for poster in topic.posters %}
+                                        <img class="img-circle" width="25" height="25"
+                                        src="{{ poster.avatar_url.format(size=50) }}"
+                                        alt="{{ poster.name }}" />
+                                        {% endfor %}
+                                    </a>
+                                </td>
+                                <td><a href="{{topic.url}}">{{topic.created_at|datetimeformat}}</a></td>
```

Comment:
Inconsistency in spaces around `{{` and `}}` maybe we should vote for one format and stick with it. I'm in favor of `{{ topic.url }}` (with spaces), the only drawback is within Python's `format()` syntax as we seen the other day…